.github/workflows/test.yml

```diff
@@ -23,10 +22,12 @@ jobs:
     timeout-minutes: 5
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          persist-credentials: false
       - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
         with:
           go-version-file: 'go.mod'
-          cache: true
```

Comment:
Disabling the cache for non-release builds should not be needed I think ([docs](https://woodruffw.github.io/zizmor/audits/#cache-poisoning)).

Reply:
yes, it's not critical, I just want to remove every possible tricky place because we do not build this repo every day and it does not harm us if we wait +1m to download fresh dependencies ;) 
wdyt?